<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddForm</class>
 <widget class="QWidget" name="AddForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>671</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Engine Cylinders: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textCylinders"/>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Engine Volume: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textVolume"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Tire Mark: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxTieMark">
        <item>
         <property name="text">
          <string>S</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>H</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QRadioButton" name="radioButtonTransmission">
      <property name="text">
       <string>Manual Transmission</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Car Number: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textNumber"/>
      </item>
     </layout>
    </item>
    <item row="2" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Years Of Maintenance: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="textYears"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="buttonText">
      <property name="text">
       <string>Add To Text File</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buttonBin">
      <property name="text">
       <string>Add To Bin File</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
